homeassistant/components/wirelesstag/binary_sensor.py

```diff
@@ -100,7 +100,7 @@ def __init__(self, api, tag, sensor_type):
         super().__init__(api, tag)
         self._sensor_type = sensor_type
         self._name = f"{self._tag.name} {self.event.human_readable_name}"
-        self._attr_unique_id = f"{self.tag_id}_{self._sensor_type}"
```

Comment:
As this changes the unique ID of entities, do we need a migration for the existing entities?

Reply:
Ideally, yes. I dont know how to implement it. If you could point me on documentation or example - I will implement it.

Reply:
Added migration from old id to new one. Thanks for pointing on this.